codegen/src/test/java/io/improbable/keanu/codegen/python/GetNonExportedVertices.java

```diff
@@ -0,0 +1,34 @@
+package io.improbable.keanu.codegen.python;
+
+import io.improbable.keanu.annotation.ExportVertexToPythonBindings;
+import io.improbable.keanu.vertices.Vertex;
+import org.reflections.Reflections;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Modifier;
+import java.util.Set;
+
+public class GetNonExportedVertices {
```

Comment:
It was a useful tool for me to be able to print those vertices that aren't exported to Python.  I'd say it's worth having in the repo just in case people want to do the same in future (it's certainly quicker than checking by hand if you're looking at more than a couple of vertices).  Later I'd envisage this moving to be a full test case in the same way we have one for Saveability.